app/routes.py

```diff
@@ -1,2 +1,51 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+
+class Planet:
+    def __init__(self, id, name, description, association):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.association = association
+
+    def make_planet_dict(self):
+        return dict(
+            id=self.id,
+            name=self.name,
+            description=self.description,
+            association=self.association)
```

Comment:
Stylistically, consider leaving a trailing `,` after the final argument, which makes it easier to add additional arguments later if needed. I also like to wrap the closing paren to match the indentation of the line where the open paren appeared.
```py
            association=self.association,
        )
```